tests/Feature/Http/Livewire/Validators/MonitorTest.php

```diff
@@ -300,42 +302,6 @@ function forgeBlock(string $address, int $height): void
             ]);
     });
 
-    it('should reload on new block event', function () {
-        $this->travelTo(Carbon::parse('2024-02-01 14:00:00Z'));
-
-        $this->freezeTime();
-
-        [0 => $validators] = createRealisticRound([
-            [
-                ...array_fill(0, 4, true),
-                false,
-                ...array_fill(0, 46, true),
-            ],
-            [
-                ...array_fill(0, 4, true),
-                false,
-                ...array_fill(0, 46, true),
```

Comment:
@alexbarnsley As far as I can tell this test is no longer valid there is no `echo:blocks,NewBlock` listener (as we have on the `app/Http/Livewire/BlockTable.php` component for example)

Ill let you know because I was unable to find were this test was added or if the listener was there at any point, Im assuming is not used anymore because we use polling, still double check shouldn't hurt

Reply:
I don't recall that being intentionally removed, I'll take a look after my current task :oleglala: